x-pack/test/api_integration/apis/management/remote_clusters/remote_clusters.js

```diff
@@ -109,7 +109,7 @@ export default function ({ getService }) {
     });
 
     describe('List', () => {
-      it('should return an array of remote clusters', async () => {
+      it.skip('should return an array of remote clusters', async () => {
```

Comment:
__question:__ did we file issue to re-enable this test again?

Reply:
Yes there is an issue on CCR to investigate: https://github.com/elastic/kibana/issues/27616

Reply:
Cool, thanks :+1:

Reply:
Should be back soon: https://github.com/elastic/kibana/pull/27778 😊